packages/create-llama/templates/types/simple/fastapi/pyproject.toml

```diff
@@ -12,6 +12,7 @@ uvicorn = { extras = ["standard"], version = "^0.23.2" }
 llama-index = "^0.9.19"
 pypdf = "^3.17.0"
 python-dotenv = "^1.0.0"
+pymongo = "^4.6.1"
```

Comment:
Similar to modifying the package.json, I think we should programmatically modify the TOML file (in this case, adding pymongo if `mongo` is selected as vector DB). 
https://github.com/squirrelchat/smol-toml looks like a good solution for parsing and stringifying TOML files